base/applications/mscutils/eventvwr/eventvwr.c

```diff
@@ -4007,11 +4007,79 @@ InitPropertiesDlg(HWND hDlg, PEVENTLOG EventLog)
     }
 }
 
+static
+VOID
+SavePropertiesDlg(HWND hDlg, PEVENTLOG EventLog)
+{
+    LPWSTR lpLogName = EventLog->LogName;
+
+    DWORD dwType;
+    DWORD dwMaxSize = 0, dwRetention = 0;
+    HKEY hLogKey;
+    WCHAR *KeyPath;    
+    SIZE_T cbKeyPath;
+
+    if (EventLog->Permanent)
+    {
+
```

Comment:
Extra unnecessary space. Also the whole conditional body code needs an extra indent.

Reply:
He could avoid that by just doing:
```c
if (!EventLog->Permanent)
    return;
```

Reply:
OK